book/book.json

```diff
@@ -1,4 +1,5 @@
 {
+  "gitbook": "2.x.x",
```

Comment:
This is a very different change than the issue raised at #326...

This change would require testing that I do not have time to do right now. It deserves its own PR, supported with a list of resolved issues, regressions, and test results.